sources/packages/web/public/index.html

```diff
@@ -4,14 +4,10 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width,initial-scale=1.0">
-    <meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate" />
```

Comment:
Any particular reason to remove these lines now? Would it bring back the cache issue that we faced a few days ago?

Reply:
Got the line back